Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -199,6 +199,16 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
           ``--cni=cilium`` parameter in ``minikube start`` command. However, this may not
           install the latest version of cilium.
 
+    .. group-tab:: Rancher Desktop
+
+       Install Rancher Desktop >= v1.1.0 as per Rancher Desktop documentation:
```

Comment:
I was having 1.0.1 locally, just curious why this version is not working.

Reply:
oh thanks, no wonder i used to have issue invalid flag --disable...

Reply:
oh thanks, no wonder I was having weird issue invalid flag (e.g. --disable-network-policy, etc)